pkg/services/accesscontrol/scope.go

```diff
@@ -0,0 +1,25 @@
+package accesscontrol
+
+import (
+	"fmt"
+	"strings"
+)
+
+// Scope builds scope from parts
+// e.g. Scope("users", "*") return "users:*"
+func Scope(parts ...string) string {
```

Comment:
This is something we can tweak around with, but do we want a Scope type? That'd make parameter expansion and things like that easier 🙂

Reply:
(not for this PR)